charts/kubernetes-agent/templates/tentacle-deployment.yaml

```diff
@@ -46,6 +46,10 @@ spec:
               value: {{ join "," .Values.agent.targetEnvironments | quote }}
             - name: "TargetRole"
               value: {{ join "," .Values.agent.targetRoles | quote }}
+            {{- with .Values.agent.machinePolicy }}
+            - name: "MachinePolicy"
+              value: {{ . | quote }}
+            {{- end }}
```

Comment:
Is this env var already supported in the docker image?

Reply:
Yep 👍 Tested and it works with just this change